drivers/ieee802154/Kconfig

```diff
@@ -92,6 +92,13 @@ config IEEE802154_CSL_DEBUG
 	help
 	  Enable support for CSL debugging by avoiding sleep state in favor of receive state.
 
+config IEEE802154_SELECTIVE_TXCHANNEL
+	bool "Support for selective TX channel setting"
+	help
+	  Enable support for selectively setting TX channel for every timed transmission request.
+	  The drivers MAY have the capability IEEE802154_HW_SELECTIVE_TXCHANNEL if and only if
```

Comment:
I can see you assume the capability may not be set even if this Kconfig is enabled, so not sure if "if and only if" shouldn't be "only if" :).

Reply:
The capability MAY be set if the Kconfig is set. Otherwise it MUST NOT be set.

Reply:
rephrased